### PR TITLE
Rowan/feature/prints to logs tt

### PR DESCRIPTION
**Pull request by rrasmuss4200**
Added logging for BulkMsgDisp, ComsHandler and DFGM_handler. Logs are created in respective process' directories and can be downlinked

### CODE REVIEW

**Comment by rrasmuss4200** [comment]
To review, just pull the changes into my branch on your local device and make sure everything runs properly. I.e, you can uplink a command and downlink data

File: ex3_ground_station/cli_ground_station/src/main.rs
Comment (kulovac) [inline ex3_ground_station/cli_ground_station/src/main.rs:1]: More of an aside, there are a lot of unused imports and commented-out code here

File: ex3_shared_libs/logging/log4rs.yml
Comment (kulovac) [inline ex3_shared_libs/logging/log4rs.yml:1]: Most of the logging files stop just above 4kB, is that intentional? The log files for the coms and dfgm stop around ~4100B

**Review by kulovac — APPROVED** [review]
Logs work for IRIS, DFGM, Coms. Looks good. Could be worth going over the max file size for the logs given that IRIS does go to 4500B.

File: ex3_ground_station/cli_ground_station/src/main.rs
Comment (rrasmuss4200) [reply ex3_ground_station/cli_ground_station/src/main.rs:1]: Ya, not too worries about those for now. They can be easily cleaned up

**Pull request merged**